packages/volto-slate/src/blocks/Text/SlashMenu.jsx

```diff
@@ -110,7 +114,10 @@ const PersistentSlashMenu = ({ editor }) => {
       )
         .filter(
           // TODO: make it work with intl?
-          (block) => slashCommand && block.id.indexOf(slashCommand[1]) === 0,
+          (block) =>
+            slashCommand &&
+            (block.id.indexOf(slashCommand[1]) === 0 ||
+              block.title.toLowerCase().indexOf(slashCommand[1]) === 0),
```

Comment:
@danalvrz Try typing upper and lower case forms of both ids and titles. I think you need to lower case both the values from the block data and the text that the user typed, before comparing.